### PR TITLE
wxRadioButton, navigation among siblings

### PR DESCRIPTION
**Pull request by csomor**
The existing functionality is exposed via base class methods

### CODE REVIEW

File: include/wx/msw/radiobut.h [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 #include "wx/msw/ownerdrawnbutton.h"
 
-class WXDLLIMPEXP_CORE wxRadioButton : public wxMSWOwnerDrawnButton<wxControl>
+class WXDLLIMPEXP_CORE wxRadioButton : public public wxRadioButtonBase<wxMSWOwnerDrawnButton<wxControl>>
```
Comment (vadz) [inline include/wx/msw/radiobut.h:16]:
```suggestion
class WXDLLIMPEXP_CORE wxRadioButton : public wxRadioButtonBase< wxMSWOwnerDrawnButton<wxControl> >
```
(there are 2 changes here: fix the extra `public` and add a space for C++98 compatibility)

**Review by vadz — COMMENTED** [review]
Thanks, no real comments, but we need to fix the build, of course, and we also need to document the new functions in `interface/wx/radiobut.h`. Ideal would be to also add some unit tests for them, but I guess we can live without them as this is not really new code...

File: include/wx/msw/radiobut.h (hunk 1 above)
Comment (csomor) [reply include/wx/msw/radiobut.h:16]: > Thanks, no real comments, but we need to fix the build, of course, and we also need to document the new functions in `interface/wx/radiobut.h`. Ideal would be to also add some unit tests for them, but I guess we can live without them as this is not really new code...

Thanks, yes, I overlooked the Win builds ...

**Comment by csomor** [comment]
@vadz, I've added simple one-liners, do we need more ?

**Comment by vadz** [comment]
Sorry, I don't see the updates, haven't you actually pushed them yet?

In any case, there is not much to say about these methods, but I'd at least mention the behaviour for `wxRB_SINGLE` case.

**Comment by csomor** [comment]
sorry, of course, done now, and I've added wxRB_SINGLE

**Comment by vadz** [comment]
Sorry, I don't think the documentation of `wxRB_SINGLE` case is correct -- I asked about it because I wasn't sure, but looking at it now, I see that it actually returns the button itself from `Get{First,Last}` in this case. I'll fix this in a moment.

**Comment by csomor** [comment]
yes, your right, sorry, I overlooked this completely

**Comment by vadz** [comment]
Trying to add a test for these functions I've also discovered that they currently can't be used (the test doesn't link) under Linux, so I'm going to fix this too, but it will take a bit longer...

**Comment by vadz** [comment]
I couldn't prevent myself from making a few more improvements, but I'm finally done here, I think, so if the CI builds pass and if you don't have any objections, I'm going to merge this soon.

**Comment by csomor** [comment]
thanks a lot

**Comment by vadz** [comment]
Sorry, no, after looking at all this once again, I've realized that we could avoid having a template `wxRadioButtonBase` rather easily, after all, so I've decided to simplify the code, especially because it also allows to put the code directly in `wxRadioButtonBase` methods instead of using extra functions.

But **this** one is really the final version, promised.

**Pull request merged**